README.md

```diff
@@ -32,8 +32,7 @@ Variables to control how to configure the proxy (can be set per location, see
 (Files must end in .rules to be loaded)
 * `NAXSI_RULES_MD5_CSV` - A CSV of md5 hashes for the files specified above
 * `NAXSI_USE_DEFAULT_RULES` - If set to "FALSE" will delete the default rules file.
-* `ENABLE_UUID_PARAM` - If set to "FALSE", will NOT add a UUID header to all requests. If set to HEADER will add this
- for easy tracking in down stream logs e.g. `X-Request-Id: 50c91049-667f-4286-c2f0-86b04b27d3f0`.
+* `UUID_VALUE` - The value of the UUID to add as an `X-Request-Id` header to proxied requests. Default is `$request_id`.
```

Comment:
some improved wording, and also consider changing the variable name (you'd have to change it everywhere if you do that).

```suggestion
* `UUID_VARIABLE_NAME` - The name of the variable used to set `$uuid` which is in turn used to populate the `X-Request-Id` header sent to proxied services. Default is `$request_id` which is automatically generated by [ngx_http_core_module](http://nginx.org/en/docs/http/ngx_http_core_module.html). Another option is $http_x_request_id which would represent the `X-Request-ID` header from the incoming request.
```

Reply:
(I found UUID_VALUE confusing because I was expecting it to have the value of the uuid, rather than a variable name)